types/mongodb/index.d.ts

```diff
@@ -1953,24 +1968,34 @@ export interface FindAndModifyWriteOpResultObject<TSchema> {
 }
 
 /** http://mongodb.github.io/node-mongodb-native/3.1/api/Collection.html#findOneAndReplace */
-export interface FindOneAndReplaceOption extends CommonOptions {
-    projection?: object;
-    sort?: object;
+export interface FindOneAndReplaceOption<T> extends CommonOptions {
+    projection?: SchemaMember<T, ProjectionOperators | number | boolean | any>;
```

Comment:
Why is `any` allowed here?

According to https://docs.mongodb.com/manual/reference/method/db.collection.find/#find-projection, only `0`, `1`, `true`, `false` and the `ProjectionOperators` are allowed.

Reply:
That was originally what I thought :D 
But @HitkoDev notice me the MongoDB 4.4 introduced aggregation expression in projection(https://docs.mongodb.com/manual/reference/method/db.collection.find/#find-projection), for example: `{maxInArray: {$max: ['$array', 0]}}`, and it's hard to typing the aggregation expression and maybe some operator not support in projection. 

So for now I added `any` to projection, or have a better solution?